Package.swift

```diff
@@ -8,9 +8,8 @@
 // A user of the Swift Package Manager (SPM) package will consume this file directly from the ORT SPM github repository.
 // For context, the end user's config will look something like:
 //
-// TODO: UPDATE VERSION NUMBER HERE ONCE A 1.16.0 RELEASE IS CONDUCTED ON THE SPM REPO
 //     dependencies: [
-//       .package(url: "https://github.com/microsoft/onnxruntime-swift-package-manager", from: "1.15.0"), 
+//       .package(url: "https://github.com/microsoft/onnxruntime-swift-package-manager", from: "1.16.0"), 
```

Comment:
I suppose we won't need to keep updating this comment every ORT release as it's a usage example, right?

Reply:
Or maybe add something to the comment pointing them to a location where they can find the valid release versions including the latest. 

Maybe searching on [https://cocoapods.org/](https://cocoapods.org/) for onnxruntime-c, however there seems to be some sort of publishing issue as it's only showing 1.14 as the latest version. The podspecs exist though: [https://github.com/CocoaPods/Specs/tree/master/Specs/3/a/a/onnxruntime-c](https://github.com/CocoaPods/Specs/tree/master/Specs/3/a/a/onnxruntime-c)

Or point to [https://github.com/microsoft/onnxruntime/releases](https://github.com/microsoft/onnxruntime/releases), although we don't always release a patch version on all platforms so that also has potential issues.

Reply:
how about the releases for this repo?
https://github.com/microsoft/onnxruntime-swift-package-manager/releases

Reply:
added comment.